lib/go-tc/traffic_monitor.go

```diff
@@ -70,7 +70,7 @@ type TMCacheGroup struct {
 	Coordinates MonitoringCoordinates `json:"coordinates"`
 }
 
-// Coordinates ...
+// MonitoringCoordinates ...
```

Comment:
again, this is technically fine but could use a real description.

Reply:
Added a description in df9e2b26f8.